content/_help/manage-your-account/delete-your-account._en.md

```diff
@@ -45,5 +44,5 @@ As a security measure, Login.gov requires a two-step process and 24 hour waiting
 5. Read through all the information carefully to make sure deleting your account is your only option
 6. Click on “Yes, continue deletion”
 7. You will receive two emails.
-        *The first email confirms we received your request. Your account is not yet deleted. Additional action is required.*
-        The second email is sent to you 24 hours later. Follow the directions in that email to complete the deletion process.
\ No newline at end of file
+* The first email confirms we received your request. Your account is not yet deleted. Additional action is required.
```

Comment:
I'm surprised you didn't need 2-4 [spaces of indenting to make it a sub-list](https://commonmark.org/help/tutorial/10-nestedLists.html) of the 7th numbered item, but sure enough it renders fine in the preview site. It doesn't render correctly on the [GitHub Markdown preview](https://github.com/18F/identity-site/blob/dbe8de2c739c510477e80519e8a42a99731a277a/content/_help/manage-your-account/delete-your-account._en.md) so maybe we want to adjust it to avoid any ambiguity, but not a blocker.